pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtl.java

```diff
@@ -285,7 +300,13 @@ static Pipeline buildHapiJdbcFetch(
       if (numResources == 0) {
         continue;
       }
+
       foundResource = true;
+      PCollection<KV<String, Long>> initialPCollection =
```

Comment:
nit: Please add a TODO in `runFhirFetch` to add this in that function too such that this approach works when we use FHIR-search mode.

Reply:
Done